forest/cli/src/cli/shutdown_cmd.rs

```diff
@@ -0,0 +1,28 @@
+// Copyright 2019-2023 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use forest_rpc_client::common_shutdown;
+
+use super::{handle_rpc_err, Config};
+use crate::cli::prompt_confirm;
+
+#[derive(Debug, clap::Args)]
+pub struct ShutdownCommand {
+    /// Assume "yes" as answer to shutdown prompt
+    #[arg(long, short)]
+    yes: bool,
```

Comment:
Let's keep it to `force` like in
https://github.com/ChainSafe/forest/blob/main/forest/cli/src/cli/snapshot_cmd.rs#L93-L95

Reply:
Sure, will do!